service/src/test/java/greencity/service/HabitStatisticServiceImplTest.java

```diff
@@ -239,8 +239,8 @@ void getAmountOfAcquiredHabitsByUserIdTest() {
     @Test
     void testCalculateUserInterest() {
         when(userRemoteClient.countActiveUsers()).thenReturn(100L);
-        when(habitRepo.countActiveHabitCreators()).thenReturn(List.of(1L, 2L, 3L));
-        when(habitRepo.countActiveHabitFollowers()).thenReturn(List.of(4L, 5L));
+        when(habitRepo.countActiveHabitCreators(anyList())).thenReturn(List.of(1L, 2L, 3L));
+        when(habitRepo.countActiveHabitFollowers(anyList())).thenReturn(List.of(4L, 5L));
```

Comment:
```suggestion
        List<Long> activatedUserIds = List.of(3L, 7L, 11L);
        when(userRemoteClient.getActivatedUsersIds())
                .thenReturn(activatedUserIds);
        when(habitRepo.countActiveHabitCreators(activatedUserIds))
                .thenReturn(List.of(1L, 2L, 3L));
        when(habitRepo.countActiveHabitFollowers(activatedUserIds))
                .thenReturn(List.of(4L, 5L));
```

I believe it is always better to avoid 'any' calls and be more precise when there is such possibility